internal/providers/terraform/azure/vpn_gateway_connection_test.go

```diff
@@ -0,0 +1,16 @@
+package azure_test
+
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+)
+
+func TestAzureRMVpnGatewayConnection(t *testing.T) {
+	t.Parallel()
```

Comment:
```suggestion
```
We need to remove t.Parallel testing because this test outputs log warnings that now get captured in the golden file ("No products found for azurerm_virtual_network_gateway.Basic VPN gateway P2S tunnels (over 128), using 0.00")